tests/Cucumber/CompatibilityTest.php

```diff
@@ -127,7 +138,11 @@ public static function badCucumberFeatures(): iterable
      */
     private static function getCucumberFeatures(string $folder): iterable
     {
-        foreach (new FilesystemIterator(self::TESTDATA_PATH . $folder) as $file) {
+        $fileIterator = new FilesystemIterator(self::TESTDATA_PATH . $folder);
+        /**
+         * @var iterable<string, SplFileInfo> $fileIterator
```

Comment:
isn't phpstan smart enough to infer it ?

Reply:
I'll double check, but I think I had to do it specifically because it couldn't.

If I may hazard a guess, maybe there might be something missing/wrong in the PHPStan stubs.